### PR TITLE
fix(css): for today sections

### PR DESCRIPTION
**Pull request by gepifanio**
# Description

Fixes: https://todaydesign.atlassian.net/browse/UOM18B-217

[Standard](https://deploy-preview-1675--pattern-lib-unimelb.netlify.app/?utm_source=github&utm_campaign=bot_dp&selectedKind=Today%2FSection&selectedStory=Standard%20%28with%20columns%29&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)
[Standard with Image](https://deploy-preview-1675--pattern-lib-unimelb.netlify.app/?utm_source=github&utm_campaign=bot_dp&selectedKind=Today%2FSection&selectedStory=Standard%20%28with%20image%29&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)
[3 up](https://deploy-preview-1675--pattern-lib-unimelb.netlify.app/?utm_source=github&utm_campaign=bot_dp&selectedKind=Today%2FSection&selectedStory=3%20Up&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)
[Columns](https://deploy-preview-1675--pattern-lib-unimelb.netlify.app/?utm_source=github&utm_campaign=bot_dp&selectedKind=Today%2FSection&selectedStory=Columns&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)
[Testimonial](https://deploy-preview-1675--pattern-lib-unimelb.netlify.app/?utm_source=github&utm_campaign=bot_dp&selectedKind=Today%2FSection&selectedStory=Testimonials%20%28text%29&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)
[Testimonials Long](https://deploy-preview-1675--pattern-lib-unimelb.netlify.app/?utm_source=github&utm_campaign=bot_dp&selectedKind=Today%2FSection&selectedStory=Testimonials%20%28long%20text%29&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)
[Testimonials video](https://deploy-preview-1675--pattern-lib-unimelb.netlify.app/?utm_source=github&utm_campaign=bot_dp&selectedKind=Today%2FSection&selectedStory=Testimonials%20%28video%29&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)
[Bordered](https://deploy-preview-1675--pattern-lib-unimelb.netlify.app/?utm_source=github&utm_campaign=bot_dp&selectedKind=Today%2FSection&selectedStory=Bordered&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)
[Card article large](https://deploy-preview-1675--pattern-lib-unimelb.netlify.app/?utm_source=github&utm_campaign=bot_dp&selectedKind=Today%2FCards&selectedStory=Article%20Large&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)

# Checklist:

- [x] Make sure to do not repeat yourself (DRY)
- [x] Snapshots tested
- [x] A11y tested
- [x] CSS utilises BEM naming convention and structure
- [x] Code does not mutate variables/objects/arrays but instead uses `map`, `filter` etc to return a new array/object
- [x] Crossbrowser testing (Safari 8+, iOS 8.4+, Android 4.4+, Firefox ESR (v52.x), iPhone (4s,6), iPad 2, Galaxy s5) 
- [x] My changes generate no new warnings
- [x] I have added tests (e2e and unit) that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Added the new component to the index.js (Vue and Lib) export

# Reviewer Checklist

- [ ] Code is not repeated (DRY)
- [ ] ES6+ code is used where possible
- [ ] Code does not mutate variables/objects/arrays but instead uses `map`, `filter` etc to return a new array/object
- [ ] CSS utilises BEM naming convention and structure
- [ ] Snapshots tested against a copy of dev snapshots to check UI changes are intended
- [ ] A11y tested
- [ ] Crossbrowser tested in at least IE11

### CODE REVIEW

**Comment by angieraj** [comment]
Thanks @gepifanio 

1. Standard - Looks good on smaller break points
2. Standard with Image - Looks good on smaller break points
3. 3 up - Starts to get a little crunched between 805px width and 605px width. Not sure this is an issue though. Thoughts @jaredhenderson 
4. Columns - Looks good on smaller break points
5. Testimonial - Image looks quite big on 581px width and 482px width, again will defer to @jaredhenderson on this 
6. Testimonials Long - Looks good on smaller break points
7. Testimonials video - Looks good on smaller break points
8. Bordered - Looks good on smaller break points
9. Card article large - Looks good on smaller break points

File: components/section/_section-alt.css [hunk 1]
```diff
@@ -101,7 +101,6 @@
 
   &__left {
     position: relative;
```
Comment (jaredhenderson) [inline components/section/_section-alt.css:103]: ✅

File: components/testimonials/_testimonials-alt.css [hunk 2]
```diff
@@ -5,12 +5,17 @@
   height: 100%;
   width: 100%;
   padding: 0;
-  max-width: 20rem;
   min-width: 12rem;
   margin: 0;
 
+  @media (--bp-desktop) {
+    max-width: 20rem;
```
Comment (jaredhenderson) [inline components/testimonials/_testimonials-alt.css:12]: ✅

File: components/testimonials/_testimonials-alt.css [hunk 3]
```diff
@@ -5,12 +5,17 @@
...
+  }
+
   &--long {
-    max-width: 34rem;
+    @media (--bp-desktop) {
```
Comment (jaredhenderson) [inline components/testimonials/_testimonials-alt.css:16]: ✅

File: components/_utils.css [hunk 4]
```diff
@@ -540,7 +540,9 @@ a[target='_blank'] {
  * Content max-width wrapper
  */
 .content-max-width {
-  max-width: 28rem;
+  @media (--bp-desktop) {
```
Comment (jaredhenderson) [inline components/_utils.css:543]: ✅

**Review by jaredhenderson — CHANGES_REQUESTED** [review]
Current CSS changes look good, but there could be some refinements to related parts of these components pertaining to breakpoints:
- 3 up - agreed @angieraj that these look a little squashed at the lower end of the breakpoints before switching to single column. However this is part of the base grid system and widely used by many other components, so I can't suggest a quick fix here. Would need a comprehensive review of impact and changes required

- [x] Testimonial - agreed, image is too big when changing to single column and is not restrained in width until the 480px breakpoint - adjust the image width restraint to match the column breakpoint change
Screen Shot 2021-06-24 at 12 14 09 pm

- Testimonial video - @angieraj I hadn't noticed that the behaviour for this one was to scroll off the edge on mobile? Was that the intention? Again, I can't find any reference to the intended design in Figma
Screen Shot 2021-06-24 at 12 16 25 pm

- [ ] Card article large changes to single column too soon, resulting in a huge picture from 768px to 600px. Can we adjust the column change breakpoint to 599px to see what that looks like
Screen Shot 2021-06-24 at 12 20 28 pm

**Comment by angieraj** [comment]
@jaredhenderson with the testimonial video, I believe this is intentional and was part of the template that Lucas built, will need to double check it with him. 

Jacob sent across the Figma link https://www.figma.com/file/yK135x6z5006Y7m6y2pT7H/UoM-18-Study?node-id=3442%3A35934

However it doesn't seem to have any reference to mobile versions of the testimonial video or the accordions.

**Comment by angieraj** [comment]
@jaredhenderson and @gepifanio Just wanted to flag something on 

1. Study Header [https://deploy-preview-1675--pattern-lib-unimelb.netlify.app/?utm_source=github&utm_campaign=bot_dp&selectedKind=Today%2FHeader&selectedStory=Study&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel](url)
2. [Pathfinder Slot ](https://deploy-preview-1675--pattern-lib-unimelb.netlify.app/?utm_source=github&utm_campaign=bot_dp&selectedKind=Today%2FPathfinder&selectedStory=Slot&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)

On my iPhone 11, the magnifying glass next to the input field seems to be separated from the input field. Using the responsive tool on chrome it looks fine but this is what I'm seeing. 

Home - The University of Melbourne
IMG_3035

Can we please investigate and fix that?

**Comment by jaredhenderson** [comment]
@angieraj as discussed, we will address the search box "rounded" issue separately from this one if you wouldn't mind making a ticket for it

File: components/testimonials/_testimonials-alt.css [hunk 5]
```diff
@@ -88,10 +93,6 @@
     background-color: rgb(var(--col-grey-light));
     display: block;
 
-    @media (--bp-max-mobile) {
```
Comment (jaredhenderson) [inline components/testimonials/_testimonials-alt.css:91]: - This one needs to come in at the 599px breakpoint, not be deleted

**Review by jaredhenderson — CHANGES_REQUESTED** [review]

File: components/testimonials/_testimonials-alt.css (hunk 5 above)
Comment (jaredhenderson) [reply components/testimonials/_testimonials-alt.css:91]: Not **above** 599px, **below**. We're talking about the display on mobile

File: components/cards/_card-article-large.css [hunk 6]
```diff
@@ -5,7 +5,7 @@
     display: flex;
     flex-direction: column-reverse;
 
-    @media (--bp-desktop) {
+    @media (--bp-x-tablet) {
```
Comment (jaredhenderson) [inline components/cards/_card-article-large.css:8]: This is behaving really weirdly when resizing now. Row should change to column at 599px only

**Review by jaredhenderson — CHANGES_REQUESTED** [review]

**Review by jaredhenderson — CHANGES_REQUESTED** [review]
Ok, `card article large` doesn't work well changing at 599px (the text column is too tall compared to the image) - can you please revert the changes to `_card-article-large.css` and we'll just leave it as it was

**Review by jaredhenderson — APPROVED** [review]

**Pull request merged**